package.json

```diff
@@ -24,7 +24,8 @@
     "test:ci": "ng test --single-run",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "precommit": "run-s lint test:ci e2e"
+    "precommit": "run-s lint test:ci e2e",
+    "populate-db": "ts-node ./scripts/populate-firebase.ts"
```

Comment:
I would prefer _db:populate_ as the script name. This way we could eventually add more db related scripts using the _db:sometask_ schema.